.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+	"name": "Orchid",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+	"features": {
+		"ghcr.io/devcontainers/features/docker-outside-of-docker:1": {}
+	},
+	"forwardPorts": [
+		5432,
+		5433
+	],
+	"postCreateCommand": "npm i -g pnpm",
+	"remoteUser": "node",
+	"remoteEnv": {
+		"POSTGRES_DB": "orchid-orm",
+		"POSTGRES_USER": "postgres",
+		"POSTGRES_HOST_AUTH_METHOD": "trust",
+		"MYSQL_ROOT_PASSWORD": "root",
+		"MYSQL_DATABASE": "orchid-orm"
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"GitHub.vscode-pull-request-github"
+			]
+		}
+	}
+}
```

Comment:
As you can see, tab characters don't look good in GitHub. Prettier would automatically replace tabs here with spaces if it was configured.

I'm not sure how to enforce Prettier rules to make it comfortable for future contributors: I prefer it to run automatically on save, some people find it to be annoying, some have pre-commit hooks to run eslint/prettier, some say it's better to be done in CI.

What do you think?

Reply:
I'm just using what vscode is using for formatting json. It won't let me do anything else, and I don't see any option to change it to spaces.

Reply:
@romeerez People will inevitably use different IDEs. I think the best way is `husky` + `lint-staged` (and then ideally run a linting **test** in the CI pipeline to make sure the pre-commit hook was not somehow ignored.)

By the way, if you merge the `.editorconfig` PR from #119, GitHub will start displaying tabs with the configured width automatically based on the `indent_size` setting.

Reply:
Absolutely, it's a good time to configure husky with lint-staged and CI checks.